src/core_plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -1,16 +1,223 @@
 export const mappings = {
+  '_default_': {
+    'dynamic': 'false'
+  },
+  dashboard: {
```

Comment:
👍  working on this one now.  The plan is to expose a registerMappings method on the elasticsearch plugin.  I was concerned there could be timing issues, but it looks like it will work from the plugins init method.